types/node/fs.d.ts

```diff
@@ -2788,17 +2811,18 @@ declare module 'fs' {
         filename: PathLike,
         options:
             | {
+                  bigint?: boolean | undefined;
```

Comment:
The `bigint` option being set changes the types in the `Stats` objects to be `bigint`s - shouldn't the types be updated to respect that?

Reply:
Yes, good catch! Just committed a fix.